### DISCUSSION TITLE
이름이 길때

### DISCUSSION THREAD

**Discussion by jihwanKo** [metamong-friends/metamong-friends.github.io#26] [제보하기]
![image](https://github.com/metamong-friends/metamong-friends.github.io/assets/49700905/b31e7df3-0a45-4483-8054-5de123846c77)

**Comment by ACT-TJ** [comment]
따로 추가 테스트까지 진행해 주셔셔. 감사의 말씀을 먼저 드립니다.

전체적인 정책과 기능 정리해서 업데이트 하고, 해당 이슈에 대해서도 피드백 드리도록 하겠습니다!

**Comment by shlee303** [comment]
이름 부분 글자수 제한을 추가 하고, 긴 경우 ellipsis를 추가하였습니다. 제보 감사합니다 :)
